lumen/ai/agents.py

```diff
@@ -118,28 +119,6 @@ def _steps_layout(self):
     def __panel__(self):
         return self.interface
 
-    async def _select_table(self, tables):
```

Comment:
This seemed completely unused?!

Reply:
It was for FuzzyTableMatching if none was found. There might still be a need of it if embeddings tool isn't able to find any table.

Reply:
Okay, removing for now. Can re-add later.